src/functions/template-tags/ical.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-// @TODO two functions that do the exact same thing
+// @todo [BTRIA-614]: two functions that do the exact same thing
```

Comment:
:no_entry_sign: You must use "/**" style comments for a file comment

<sub>hash: 444bb2ba49bbe71fe53848bb5ceef829</sub>